api/addon.go

```diff
@@ -7,7 +7,7 @@ import (
 	crd "github.com/konveyor/tackle2-hub/k8s/api/tackle/v1alpha1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"net/http"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	k8s "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Alias as _k8s_ to avoid name/shadowing collision with `client` variables and parameters.